wandb/internal/sender.py

```diff
@@ -193,6 +193,8 @@ def handle_stats(self, data):
         # TODO(jhr): check fs.push results?
 
     def handle_output(self, data):
+        if not self._fs:
+            return
```

Comment:
@raubitsj was getting here in tests with `self._fs` being None.  Not sure how we get into this state, I should atleast add a comment here but saw we were checking this in other methods so I added it...

Reply:
this state shouldnt happen but this is fine for now... Will have to circle back and harden some of this stuff.